src/spyglass/decoding/v0/core.py

```diff
@@ -0,0 +1,188 @@
+from spyglass.utils import logger
+
+try:
```

Comment:
Same as above, I would avoid a try/except over a whole script. If we want to avoid table declaration when a package isn't found, I think that should be a halting error